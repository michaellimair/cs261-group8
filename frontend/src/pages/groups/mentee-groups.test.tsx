import { render } from '@testing-library/react';
import MenteeGroupPage from './mentee-groups';

describe('MenteeGroupPage', () => {
  it('renders correctly', () => {
    const result = render(<MenteeGroupPage />);

    expect(result).toMatchSnapshot();
  });
});
