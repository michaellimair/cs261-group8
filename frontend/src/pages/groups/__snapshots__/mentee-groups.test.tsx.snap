// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenteeGroupPage renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div
      id="chakra-toast-portal"
    >
      <ul
        id="chakra-toast-manager-top"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-top-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-top-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-bottom-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
    </div>
    <div>
      <div
        class="chakra-stack css-uddvoq"
      >
        <div
          class="css-gmuwbf"
        >
          <h2
            class="chakra-heading css-0"
          >
            dashboard.group_meetings.title
          </h2>
        </div>
        <div
          class="chakra-stack css-uu7f4l"
        >
          <button
            class="chakra-button css-1mfapmp"
            type="button"
          >
            dashboard.group_meetings.propose_group
             
            <svg
              class="chakra-icon css-onkibi"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                fill="currentColor"
              />
            </svg>
          </button>
        </div>
        <h2
          class="chakra-heading css-0"
        >
          dashboard.group_meetings.available_groups
        </h2>
        <div
          class="css-jxw3ad"
          divider="[object Object]"
        >
          <div
            class="css-eal7pa"
          >
            <div
              class="css-70qvj9"
            >
              <div
                class="css-1m1kqp3"
              >
                <div
                  class="chakra-stack css-xtejj8"
                >
                  <h2
                    class="chakra-heading css-0"
                  >
                    Topic
                  </h2>
                  <h4
                    class="chakra-heading css-0"
                  />
                  <p
                    class="chakra-text css-zz2637"
                  >
                    Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.
                  </p>
                  <a
                    class="chakra-link css-1856l8r"
                    href="#"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                     
                    <svg
                      class="chakra-icon css-2y6psj"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <g
                        fill="none"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-width="2"
                      >
                        <path
                          d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                        />
                        <path
                          d="M15 3h6v6"
                        />
                        <path
                          d="M10 14L21 3"
                        />
                      </g>
                    </svg>
                  </a>
                </div>
              </div>
              <div
                class="css-kzfr2u"
              >
                <div
                  class="chakra-stack css-1wt1vvl"
                >
                  <h2
                    class="chakra-heading css-0"
                  >
                    19:00
                     
                    GMT
                  </h2>
                  <p
                    class="chakra-text css-1909u0b"
                  >
                    (
                    local_time
                    :
                     
                    19:00
                    )
                  </p>
                  <p
                    class="chakra-text css-z32800"
                  >
                    2 Apr 2022
                  </p>
                  <div
                    class="chakra-stack css-84zodg"
                  >
                    <p
                      class="chakra-text css-g5hezf"
                    >
                      Full Name
                    </p>
                    <span
                      class="chakra-avatar css-1ruhx4w"
                    >
                      <svg
                        aria-label=" avatar"
                        class="chakra-avatar__svg css-16ite8i"
                        role="img"
                        viewBox="0 0 128 128"
                      >
                        <path
                          d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                          fill="currentColor"
                        />
                        <path
                          d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                          fill="currentColor"
                        />
                      </svg>
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="css-1ih4w1z"
          >
            <div
              class="chakra-stack css-jj8v9o"
            >
              <button
                class="chakra-button css-5c2176"
                type="button"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="chakra-stack css-uddvoq"
    >
      <div
        class="css-gmuwbf"
      >
        <h2
          class="chakra-heading css-0"
        >
          dashboard.group_meetings.title
        </h2>
      </div>
      <div
        class="chakra-stack css-uu7f4l"
      >
        <button
          class="chakra-button css-1mfapmp"
          type="button"
        >
          dashboard.group_meetings.propose_group
           
          <svg
            class="chakra-icon css-onkibi"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
              fill="currentColor"
            />
          </svg>
        </button>
      </div>
      <h2
        class="chakra-heading css-0"
      >
        dashboard.group_meetings.available_groups
      </h2>
      <div
        class="css-jxw3ad"
        divider="[object Object]"
      >
        <div
          class="css-eal7pa"
        >
          <div
            class="css-70qvj9"
          >
            <div
              class="css-1m1kqp3"
            >
              <div
                class="chakra-stack css-xtejj8"
              >
                <h2
                  class="chakra-heading css-0"
                >
                  Topic
                </h2>
                <h4
                  class="chakra-heading css-0"
                />
                <p
                  class="chakra-text css-zz2637"
                >
                  Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.
                </p>
                <a
                  class="chakra-link css-1856l8r"
                  href="#"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                   
                  <svg
                    class="chakra-icon css-2y6psj"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <g
                      fill="none"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-width="2"
                    >
                      <path
                        d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                      />
                      <path
                        d="M15 3h6v6"
                      />
                      <path
                        d="M10 14L21 3"
                      />
                    </g>
                  </svg>
                </a>
              </div>
            </div>
            <div
              class="css-kzfr2u"
            >
              <div
                class="chakra-stack css-1wt1vvl"
              >
                <h2
                  class="chakra-heading css-0"
                >
                  19:00
                   
                  GMT
                </h2>
                <p
                  class="chakra-text css-1909u0b"
                >
                  (
                  local_time
                  :
                   
                  19:00
                  )
                </p>
                <p
                  class="chakra-text css-z32800"
                >
                  2 Apr 2022
                </p>
                <div
                  class="chakra-stack css-84zodg"
                >
                  <p
                    class="chakra-text css-g5hezf"
                  >
                    Full Name
                  </p>
                  <span
                    class="chakra-avatar css-1ruhx4w"
                  >
                    <svg
                      aria-label=" avatar"
                      class="chakra-avatar__svg css-16ite8i"
                      role="img"
                      viewBox="0 0 128 128"
                    >
                      <path
                        d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                        fill="currentColor"
                      />
                      <path
                        d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                        fill="currentColor"
                      />
                    </svg>
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="css-1ih4w1z"
        >
          <div
            class="chakra-stack css-jj8v9o"
          >
            <button
              class="chakra-button css-5c2176"
              type="button"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
